### ISSUE TITLE
Most Dump files appear to be corrupted

### ISSUE THREAD

**Issue by JackMorgan9** [cantryDev/CSGOCaseStatsViewerV2#3]
Can read all inventory history, but when analysing, most dump files are shown as corrupted and only analyses 194 cases of my 755 with all having the same error:

`java.nio.charset.MalformedInputException: Input length = 1
        at java.base/java.nio.charset.CoderResult.throwException(CoderResult.java:274)
        at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:339)
        at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
        at java.base/java.io.InputStreamReader.read(InputStreamReader.java:181)
        at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
        at java.base/java.io.BufferedReader.readLine(BufferedReader.java:326)
        at java.base/java.io.BufferedReader.readLine(BufferedReader.java:392)
        at java.base/java.nio.file.Files.readAllLines(Files.java:3333)
        at java.base/java.nio.file.Files.readAllLines(Files.java:3370)
        at de.cantry.csgocasestatsviewerv2.steam.service.AnalysisService.parseFullHistory(AnalysisService.java:58)
        at de.cantry.csgocasestatsviewerv2.steam.service.AnalysisService.analyseUnboxings(AnalysisService.java:169)
        at de.cantry.csgocasestatsviewerv2.Main.main(Main.java:42)
Dump with name cabc8c9f-e388-48fa-9aca-a0e12a575c76.dump seems to be corrupted`

**Comment by cantryDev** [comment]
Can you show how the curropted file looks like?
For example: cabc8c9f-e388-48fa-9aca-a0e12a575c76.dump

**Comment by JackMorgan9** [comment]
Sure, see the file in the attached zip.
[d98a24dd-c46b-4427-8964-e6c954b361d2.zip](https://github.com/cantryDev/CSGOCaseStatsViewerV2/files/11789267/d98a24dd-c46b-4427-8964-e6c954b361d2.zip)

**Comment by cantryDev** [comment]
![image](https://github.com/cantryDev/CSGOCaseStatsViewerV2/assets/16962714/88b15ca0-242f-4060-9a88-39e19fe4a57e)
It loads for me?
Can you replace your jar and try with this so it prints the complete error
[CSGOCaseStatsViewerV2-1.0.1-jar-with-dependencies.zip](https://github.com/cantryDev/CSGOCaseStatsViewerV2/files/11790205/CSGOCaseStatsViewerV2-1.0.1-jar-with-dependencies.zip)

**Comment by JackMorgan9** [comment]
Still seems to just give the same error message:

`java.nio.charset.MalformedInputException: Input length = 1
	at java.base/java.nio.charset.CoderResult.throwException(CoderResult.java:274)
	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:339)
	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:181)
	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
	at java.base/java.io.BufferedReader.readLine(BufferedReader.java:326)
	at java.base/java.io.BufferedReader.readLine(BufferedReader.java:392)
	at java.base/java.nio.file.Files.readAllLines(Files.java:3333)
	at java.base/java.nio.file.Files.readAllLines(Files.java:3370)
	at de.cantry.csgocasestatsviewerv2.steam.service.AnalysisService.parseFullHistory(AnalysisService.java:56)
	at de.cantry.csgocasestatsviewerv2.steam.service.AnalysisService.analyseUnboxings(AnalysisService.java:170)
	at de.cantry.csgocasestatsviewerv2.Main.main(Main.java:42)

Dump with name 1613320e-8b21-4787-8bd1-09c4d500d2e4.dump seems to be corrupted
java.nio.charset.MalformedInputException: Input length = 1
	at java.base/java.nio.charset.CoderResult.throwException(CoderResult.java:274)
	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:339)
	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:181)
	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
	at java.base/java.io.BufferedReader.readLine(BufferedReader.java:326)
	at java.base/java.io.BufferedReader.readLine(BufferedReader.java:392)
	at java.base/java.nio.file.Files.readAllLines(Files.java:3333)
	at java.base/java.nio.file.Files.readAllLines(Files.java:3370)
	at de.cantry.csgocasestatsviewerv2.steam.service.AnalysisService.parseFullHistory(AnalysisService.java:56)
	at de.cantry.csgocasestatsviewerv2.steam.service.AnalysisService.analyseUnboxings(AnalysisService.java:170)
	at de.cantry.csgocasestatsviewerv2.Main.main(Main.java:42)

Dump with name 177130c1-9109-42be-ab27-a4ccf897dbb8.dump seems to be corrupted
java.nio.charset.MalformedInputException: Input length = 1
	at java.base/java.nio.charset.CoderResult.throwException(CoderResult.java:274)
	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:339)
	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:181)
	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
	at java.base/java.io.BufferedReader.readLine(BufferedReader.java:326)
	at java.base/java.io.BufferedReader.readLine(BufferedReader.java:392)
	at java.base/java.nio.file.Files.readAllLines(Files.java:3333)
	at java.base/java.nio.file.Files.readAllLines(Files.java:3370)
	at de.cantry.csgocasestatsviewerv2.steam.service.AnalysisService.parseFullHistory(AnalysisService.java:56)
	at de.cantry.csgocasestatsviewerv2.steam.service.AnalysisService.analyseUnboxings(AnalysisService.java:170)
	at de.cantry.csgocasestatsviewerv2.Main.main(Main.java:42)

Dump with name 1a453441-8884-4b80-ba86-3418507202e1.dump seems to be corrupted`

**Comment by cantryDev** [comment]
I the loading to explicit UTF-8
Can you try again?
[CSGOCaseStatsViewerV2-1.0.1-jar-with-dependencies.jar.zip](https://github.com/cantryDev/CSGOCaseStatsViewerV2/files/11792921/CSGOCaseStatsViewerV2-1.0.1-jar-with-dependencies.jar.zip)

**Comment by JackMorgan9** [comment]
Still giving same error im afraid:

`java.nio.charset.MalformedInputException: Input length = 1
        at java.base/java.nio.charset.CoderResult.throwException(CoderResult.java:274)
        at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:339)
        at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
        at java.base/java.io.InputStreamReader.read(InputStreamReader.java:181)
        at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
        at java.base/java.io.BufferedReader.readLine(BufferedReader.java:326)
        at java.base/java.io.BufferedReader.readLine(BufferedReader.java:392)
        at java.base/java.nio.file.Files.readAllLines(Files.java:3333)
        at de.cantry.csgocasestatsviewerv2.steam.service.AnalysisService.parseFullHistory(AnalysisService.java:56)
        at de.cantry.csgocasestatsviewerv2.steam.service.AnalysisService.analyseUnboxings(AnalysisService.java:170)
        at de.cantry.csgocasestatsviewerv2.Main.main(Main.java:42)

Dump with name 0f9822b2-6ab6-4be0-8711-84f607e3b7f9.dump seems to be corrupted`

**Comment by cantryDev** [comment]
I dont understand why.
Can you try again?
If it dosent work try to dump again and try again.

[CSGOCaseStatsViewerV2-1.0.1-jar-with-dependencies.jar.zip](https://github.com/cantryDev/CSGOCaseStatsViewerV2/files/11793160/CSGOCaseStatsViewerV2-1.0.1-jar-with-dependencies.jar.zip)

**Comment by JackMorgan9** [comment]
Weirdly, after using the new jar file, deleting the data folder and the result.txt files, it has worked fine.

Worth noting I had previously deleted the data folder and still had the error, but not the result.txt files, I doubt thats the reason why it is suddenly working. I did dump again instead of trying to analyze the corrupt ones, but I have also tried dumping previously a few times after deleting the data folder and it hadn't worked.
